collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -101,6 +108,9 @@ protected void attachAd(AdAdapter adapter) {
     public void detachPlayer() {
         detachAd();
```

Comment:
food for thought for the future - can we move the ad to the `featureManager` infrastructure?